### PR TITLE
Document intended type

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
@core23 done

File: src/GoogleAuthenticator.php [hunk 1]
```diff
@@ -27,9 +27,24 @@
...
+
+    /**
+     * @var int
+     */
     protected $pinModulo;
+
+    /**
+     * NEXT_MAJOR: remove this property
+     */
     protected $fixBitNotation;
```
Comment (greg0ire) [inline src/GoogleAuthenticator.php]: lol

**Comment by greg0ire** [comment]
@sonata-project/contributors please merge

**Comment by jordisala1991** [comment]
Thanks @greg0ire !

**Pull request merged**